### ISSUE TITLE
Research Authorizations and related terms. 

### ISSUE THREAD

**Issue by hjellis** [biobanking/biobanking#39]
Updated spreadsheet to ensure that 'research authorization' is modeled similarly to 'informed consent' in regards to authorization, process, role and documentation. 

https://docs.google.com/spreadsheets/d/1p0Gtq3Id_xlJ8SFmeCGqLP-woBxhP28UFAGl7Yg363w/edit#gid=0

please request access if you need it.

**Comment by cstoeckert** [comment]
discussed on OBIB Aug 3, 2017 call. Want to model in parallel with informed consent process. Outcomes may be modeled as the result of document acts leading to a signed or unsigned document. Also consider 'authorization' (http://purl.obolibrary.org/obo/ICO_0000046) which has output 'authorization documentation' that may serve to model outcomes.

**Comment by fjmanion** [comment]
Reviewed the spreadsheet. I think the spreadsheet terms conflate an "authorization"  event and the documentation that such an event has occurred. To my mind, there is a difference between "authorization" and documentation that authorization has occurred.

Other comments:
For the term "research authorization documentation for post mortem specimen donation that is affirmative" and the converse of this in the negative; do we really need a term specific to the documentation, in addition to the outcome of the authorization event itself - the documentation should be the same in either event.

For the term "refusal to research authorization for post mortem specimen donation due to religious reasons" - I believe this would read better if the label was "refusal to give research authorization for post mortem specimen donation due to religious reasons".

For the term "research authorization for post mortem specimen doncation not obtained because legally authorized representative was afraid the decision might affect the surgery plans of donor" -- (a) there is a spelling error of "donation", and (b) I'm unclear from the definition what this term really means. Is the LAR being approached prior to an effort at life-saving surgery? If so this should probably be stated in the definition. 

I've attached a copy of the spreadsheet with comments added so you can see the comments in context. Sorry, but it's an excel file.
[Consent from BBRB.xlsx](https://github.com/biobanking/biobanking/files/1271403/Consent.from.BBRB.xlsx)

**Comment by hjellis** [comment]
thanks Frank
in response

1 and 2. . (I had to look up 'conflate' since I wasn't sure I actually knew what it means LOL) Good points, and I think we need the documentation for both, but I need to review its definition and  need to discuss with Chris
3. I edited the label to conform to similar labels, i.e. research authorization for post mortem specimen donation not obtained due to religious reasons (another example: research authorization for post mortem specimen donation not obtained because  legally authorized representative felt pressured)
4. I corrected the typo. Unfortunately that is all the information we have to go with. My impression  is the next of kin did not actually understand what was being asked of them, and they mistakenly thought that giving authorization would affect surgical plans for donor. Of course this is not the case. However, accurately reporting their (mis) perception is important for the ELSI studies on this project.
